tools/packaging/build/agent-enclave-bundle/config.json

```diff
@@ -0,0 +1,216 @@
+{
+        "ociVersion": "1.0.2-dev",
+        "process": {
+                "terminal": false,
+                "user": {
+                        "uid": 0,
+                        "gid": 0
+                },
+                "args": [
+                        "/run/rune/occlum_instance/build/bin/occlum-run",
+                        "/bin/enclave-agent",
+                        "-l",
+                        "tcp://127.0.0.1:7788"
+                ],
+                "env": [
+                        "OCCLUM_RELEASE_ENCLAVE=1",
+                        "PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+                        "TERM=xterm"
+                ],
+                "cwd": "/run/rune/occlum_instance",
+                "capabilities": {
+                        "bounding": [
+                                "CAP_AUDIT_WRITE",
+                                "CAP_KILL",
+                                "CAP_NET_BIND_SERVICE"
+                        ],
+                        "effective": [
+                                "CAP_AUDIT_WRITE",
+                                "CAP_KILL",
+                                "CAP_NET_BIND_SERVICE"
+                        ],
+                        "inheritable": [
+                                "CAP_AUDIT_WRITE",
+                                "CAP_KILL",
+                                "CAP_NET_BIND_SERVICE"
+                        ],
+                        "permitted": [
+                                "CAP_AUDIT_WRITE",
+                                "CAP_KILL",
+                                "CAP_NET_BIND_SERVICE"
+                        ],
+                        "ambient": [
+                                "CAP_AUDIT_WRITE",
+                                "CAP_KILL",
+                                "CAP_NET_BIND_SERVICE"
+                        ]
+                },
+                "rlimits": [
+                        {
+                                "type": "RLIMIT_NOFILE",
+                                "hard": 8192,
+                                "soft": 8192
```

Comment:
The value for hard and soft limits should extend to a bigger value, such as 65535. Because when pulling an image, such as Redis, it will extend the resource limit  8192.

Reply:
fixed, thanks!